### PR TITLE
Refactoring of travis tests

### PR DESCRIPTION
**Pull request by ksimon1**
travis tests are now in separate bash files.
removed unnecessary commands in makefile

Signed-off-by: ksimon1 <ksimon@redhat.com>

### CODE REVIEW

File: travis_ci/test.sh [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+mkdir -p "$PWD/pvs/$name"
+qemu-img convert -p -O raw $name "$PWD/pvs/$name/disk.img"
+sudo chown 107:107 "$PWD/pvs/$name/disk.img"
+sudo chmod -R a+X "$PWD/pvs"
+
+size_MB=$(( $(qemu-img info $name --output json | jq '.["virtual-size"]') / 1024 / 1024 + 128 ))
+# Create PV and PVC
+bash create-minikube-pvc.sh "$name" "${size_MB}M" "$PWD/pvs/$name/" | tee | oc apply -f -
+
+sizes=("tiny" "small" "medium" "large")
```
Comment (MarSik) [inline travis_ci/test.sh]: Not all OSes have to have all sizes. We moved the complicated ones to stdci though (Windows).

**Comment by ksimon1** [comment]
cc @MarSik, @fromanirh

File: travis_ci/test.sh [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2019 Red Hat, Inc.
+#
+
```
Comment (fromanirh) [inline travis_ci/test.sh]: please add `set -e` right at the beginning, so the script bail out at the first error. Do we want also to add `-x`, so the shell is automatically print out the command it is about to run?

File: travis_ci/test.sh [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+name=$1
+workload=$2
```
Comment (fromanirh) [inline travis_ci/test.sh]: please add code to handle missing parameters, it should print a short help/usage message and the exit

File: travis_ci/test_syntax.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2019 Red Hat, Inc.
+#
+
```
Comment (fromanirh) [inline travis_ci/test_syntax.sh]: same comment about  `set -e`

**Pull request merged**